include/Macro.h

```diff
@@ -560,18 +563,68 @@
 #    endif // MHD
 #  endif // FLU_SCHEME
 
+
 #elif ( MODEL == ELBDM )   // ELBDM
-#  ifdef LAPLACIAN_4TH
-#     define FLU_GHOST_SIZE         6
-#  else
-#     define FLU_GHOST_SIZE         3
+// use finite-difference scheme by default
+#  ifndef WAVE_SCHEME
+#  define WAVE_SCHEME WAVE_FD
 #  endif
 
+#  if ( WAVE_SCHEME == WAVE_FD )
+#     ifdef LAPLACIAN_4TH
+#        define FLU_GHOST_SIZE         6
+#     else
+#        define FLU_GHOST_SIZE         3
+#     endif
+#  elif ( WAVE_SCHEME == WAVE_GRAMFE )
+#        define FLU_GHOST_SIZE         8
```

Comment:
It is to some extent. In principle, the code works with arbitrary sizes and should become better with larger FLU_GHOST_SIZE. I have tested different sizes with the GaussianWavePacket test, but not with other tests. Since I can only vouch for the stability of the resulting schemes for FLU_GHOST_SIZE 6 and 8 which I have more thoroughly tested, I will leave a comment and check the value in `Aux_Check_Parameter()`.